src/components/toast/toast.scss

```diff
@@ -173,6 +173,15 @@ md-toast {
       top: 0;
     }
 
+    // Support for RTL alignment
+    &._md-start {
```

Comment:
It's a new feature, there should be two new position properties for the toast - `start` and `end`. The prefix will be automatically applied in the controller. `'md-' + positionFlag`